dmutils/email/dm_mailchimp.py

```diff
@@ -120,7 +122,16 @@ def subscribe_new_emails_to_list(self, list_id, email_addresses):
                 success = False
         return success
 
-    @timeout_retry
     def get_email_addresses_from_list(self, list_id):
```

Comment:
Why not make `pagination_size` a kwarg here with a default of 1000 instead of setting a module global?

Reply:
Yep, makes sense. I was trying to avoid having to update multiple places if the default changes, but what you've suggested does that to.